sql-join/magic-mallrats.sql

```diff
@@ -0,0 +1,11 @@
+SELECT "firstName",
+  "lastName"
+FROM "customers"
+WHERE "customerId" IN (
+  SELECT "customerId"
+FROM "rentals"
+JOIN (SELECT "inventoryId"
+  FROM "inventory"
+  JOIN "films" USING ("filmId")
+  WHERE "title" = 'Magic Mallrats') AS "m"
+  USING ("inventoryId"));
```

Comment:
This is more complex than it needs to be. You can just do a straight join on rentals, inventory, and films. No need to do the inner selects.